### ISSUE TITLE
persistence and retry behaviour for private transactions

### ISSUE THREAD

**Issue by hosie** [LF-Decentralized-Trust-labs/paladin#224]
deliberately deferred until we have a well established, efficient threading model for happy path / in-memory assembling / sequencing / endorsing etc.....  Decisions yet to be made:
 - do we even need persistence of "a private tx" just like `publictxmgr` has "a public tx"? Or is it just part of the paladin front door `txmgr`'s responsibilities
  - in the multi node scenarios (which will become clearer after #222 is delivered, do we need a persistence checkpoint at the time of handover between nodes?

Happy for this task to sit on the backlog while we do some other tasks that will put us in a better place to answer the above conversations.

**Comment by annamcallister** [comment]
Covered by the distributed sequencer in V1
